test/e2e/installplan_e2e_test.go

```diff
@@ -2206,6 +2212,7 @@ var _ = Describe("Install Plan", func() {
 					}
 					return true, nil
 				})
+				require.NoError(GinkgoT(), err)
```

Comment:
nit: since `err` is a free variable within this loop body, you can check this once after both `if` statements.

Reply:
I don't believe that this is true, if the first wait failed and had an error but the second did not the first error would be overwritten.